api/server/server.go

```diff
@@ -1393,35 +1391,27 @@ func (s *Server) postContainerExecCreate(eng *engine.Engine, version version.Ver
 	if err := parseForm(r); err != nil {
 		return nil
 	}
-	var (
-		name         = vars["name"]
-		job          = eng.Job("execCreate", name)
-		stdoutBuffer = bytes.NewBuffer(nil)
-		outWarnings  []string
-		warnings     = bytes.NewBuffer(nil)
-	)
+	var name = vars["name"]
 
-	if err := job.DecodeEnv(r.Body); err != nil {
+	execConfig := runconfig.ExecConfig{}
```

Comment:
sorry, these changes don't look important but it's good to be consistent with the go idioms.